connector/connect/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -587,6 +588,14 @@ class SparkConnectPlanner(session: SparkSession) {
       DataTypeProtoConverter.toCatalystType(cast.getCastToType))
   }
 
+  private def transformCaseWhen(casewhen: proto.Expression.CaseWhen): Expression = {
+    CaseWhen(
+      branches = casewhen.getBranchesList.asScala
+        .map(b => (transformExpression(b.getCondition), transformExpression(b.getValue))),
```

Comment:
[error] /home/runner/work/spark/spark/connector/connect/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala:594:13: type mismatch;
[error]  found   : scala.collection.mutable.Buffer[(org.apache.spark.sql.catalyst.expressions.Expression, org.apache.spark.sql.catalyst.expressions.Expression)]
[error]  required: Seq[(org.apache.spark.sql.catalyst.expressions.Expression, org.apache.spark.sql.catalyst.expressions.Expression)]
[error]         .map(b => (transformExpression(b.getCondition), transformExpression(b.getValue))),

Reply:
guessing you need a `.toSeq`?

Reply:
thanks